lib/zclient.h

```diff
@@ -978,13 +1010,15 @@ extern void zclient_send_mlag_data(struct zclient *client,
  * below to avoid sub-type collisions. Clients use the registration
  * apis to manage the specific opaque subtypes they want to receive.
  */
-int zclient_send_opaque(struct zclient *zclient, uint32_t type,
-			const uint8_t *data, size_t datasize);
+enum zclient_send_status zclient_send_opaque(struct zclient *zclient,
```

Comment:
still a couple of places in ldp_zebra.c using `int`

Reply:
yep fixed